buzzmobile/tests/process/test_bearing.py

```diff
@@ -19,15 +19,16 @@ def test_distance(self):
 @rostest_utils.with_roscore
 class TestBearingNode(unittest.TestCase):
     
-    @rostest_utils.launch_node('buzzmobile', 'bearing.py', 'params.launch')
+    @rostest_utils.with_launch_file('buzzmobile', 'params.launch')
+    @rostest_utils.launch_node('buzzmobile', 'bearing.py')
     def test_bearing_node(self):
         result = None
         def callback(data):
             nonlocal result
             result = data.data
 
-        with rostest_utils.mock_node('fix', NavSatFix, queue_size=None) as fix_node:
-            with rostest_utils.test_node('bearing', Float64, callback) as tn:
+        with rostest_utils.mock_node('/buzzmobile/fix', NavSatFix, queue_size=None) as fix_node:
+            with rostest_utils.test_node('/buzzmobile/bearing', Float64, callback):
```

Comment:
Also why are you mocking/testing these topics in the `buzzmobile` namespace? Bearing does not subscribe or publish anything in the buzzmobile namespace, as per file above.